tests/admin-unlock/main.yml

```diff
@@ -267,7 +271,10 @@
 
     - name: Fail when error message is incorrect
       fail:
-        msg: "Error message is incorrect"
+        msg: |
+          Expected: Error message should indicated the deployment is already in
+                    the unlocked state: development
+          Actual: double_unlock.stderr
```

Comment:
Do we need double curly braces here?  It probably works without them, but adding them would be consistent elsewhere.

Reply:
I missed that one!  I think it would actually print out the literal double_unlock.stderr